c.h

```diff
@@ -2305,6 +2305,10 @@ extern ROCKSDB_LIBRARY_API rocksdb_sstfilewriter_t*
 rocksdb_sstfilewriter_create_with_comparator(
     const rocksdb_envoptions_t* env, const rocksdb_options_t* io_options,
     const rocksdb_comparator_t* comparator);
+extern ROCKSDB_LIBRARY_API rocksdb_sstfilewriter_t*
+rocksdb_sstfilewriter_create_with_column_family_handle(
```

Comment:
Hi @ltagliamonte-dd 
I think you should drive your PR to upstream https://github.com/facebook/rocksdb/blob/v10.0.1/include/rocksdb/c.h instead